diff -uNr openssl-1.1.1d/Configurations/10-main.conf openssl-1.1.1d/Configurations/10-main.conf
--- openssl-1.1.1d/Configurations/10-main.conf	2019-09-10 21:13:07.000000000 +0800
+++ openssl-1.1.1d/Configurations/10-main.conf	2020-10-26 15:41:38.071704454 +0800
@@ -752,6 +752,11 @@
         perlasm_scheme   => "64",
         multilib         => "64",
     },
+    "linux64-loongarch64" => {
+        inherit_from     => [ "linux-generic64" ],
+        perlasm_scheme   => "64",
+        multilib         => "64",
+    },
 
     #### IA-32 targets...
     #### These two targets are a bit aged and are to be used on older Linux
diff -uNr openssl-1.1.1d/Configurations/20-debian.conf openssl-1.1.1d/Configurations/20-debian.conf
--- openssl-1.1.1d/Configurations/20-debian.conf	2020-10-23 11:46:21.000000000 +0800
+++ openssl-1.1.1d/Configurations/20-debian.conf	2020-10-26 10:05:40.667472069 +0800
@@ -51,6 +51,10 @@
 	"debian-ia64" => {
 		inherit_from => [ "linux-ia64", "debian" ],
 	},
+	"debian-loong64" => {
+		inherit_from => [ "linux64-loongarch64", "debian" ],
+		cflags => add("-DL_ENDIAN"),
+	},
 	"debian-m68k" => {
 		inherit_from => [ "linux-generic32", "debian" ],
 	},
